### DISCUSSION TITLE
View the findings from our community survey!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#102801] [Announcements]
We love hearing from our members and we love sharing back what we learned from you from our latest survey. Check out our findings from our October 2023 Community survey! 

![Jan 2024 Community Survey Results](https://github.com/community/community/assets/92824518/ca1d715a-6be2-4b8b-b4dc-91750c457dcd)

### Alt Text: Infographic entitled Community Survey Results

Long Description:
The infographic provides a snapshot of different data points from the GitHub Community Survey (October 2023). It has a solid black background and graphics are accented in purple, two different pink, and yellow. 

Centered at the top of the graphic is the text: "Thank you to the 150+ members who shared their candid feedback with us in our community survey. We enjoyed reading the responses and learning how we can improve!" 

There are four different graphics to correspond with four different data points and descriptions. The graphics and descriptions are as follows:

A graphic rocket that is yellow, next to a header entitled “Community Usage”, and the text "The most common way you use Community Discussions: learning about new features and updates. Closely followed by report bugs and Search for already submitted feature requests or bugs".

Below and to the left, a percentage bar that is 55% yellow and 45% pink to represent the data shared: "Value", and the text: "A little more than half of you (55%) found the community helps you get more value out of GitHub and all of you gave us ideas to help that improve!"

Then, a pink speech bubble with an off-centered yellow outline saying 39% in with paired with the text: "Have been a part of the community for three months or less!" and "If you’re new to the community be sure to check out our New Member Welcome thread to introduce yourself."

Next, a mostly white calendar with pink and yellow accents and the 26% percent figure is coupled with the header: "26% Said they visit the community everyday or more often." and the text: "This was the most common visit frequency and we’re thrilled so many of you join us daily."

Below that is the section with the yellow header "In Your Own Words" subheaded with "What you had to say!" and contain positive and negative direct quotes from survey respondents. 

Under the purple header "What You Like..." are the following quotes: “Having a (public) portal to connect to the GitHub developers to give feedback and report bugs without having to open a (private) support ticket.”, “Being able to help others with things I'm passionate about teaching.", and "“The brilliance of its members!!“. 

Under the purple header "Where We Can Improve..." are the following quotes: “More tutorials and more individuals reaching out to new members to share knowledge about how to get more involved & make contributions”, "It often feels like the community feedback is completely ignored”, and "“The feedback system”. 

The last section has a purple heading "Stay tuned" and underneath the text "We run this survey twice a year so we can continuously improve your community experience. Be on the lookout!"

Alt Title: Infographic for GitHub Community Discussions Survey (October 2023) 
File Name: Jan 2024 Community Survey Results. png

**Comment by VictorMs** [comment]
Hello,
I have private repositories, but I can't commit source code into my branch since two years ago.  when I to do submit without any error message. could you tell me the reasons please ?

**Reply by queenofcorgis** [reply]
@VictorMs you'll have a better chance of getting some help if you [create a new post in our Repos category](https://github.com/orgs/community/discussions/new?category=repositories) and share as many details as you can. Good luck!

**Comment by Andreasannoeuzg** [comment]
> We love hearing from our members and we love sharing back what we learned from you from our latest survey. Check out our findings from our October 2023 Community survey!
> 
> ![Jan 2024 Community Survey Results](https://private-user-images.githubusercontent.com/92824518/300945860-ca1d715a-6be2-4b8b-b4dc-91750c457dcd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc5MzE2MjYsIm5iZiI6MTcwNzkzMTMyNiwicGF0aCI6Ii85MjgyNDUxOC8zMDA5NDU4NjAtY2ExZDcxNWEtNmJlMi00YjhiLWI0ZGMtOTE3NTBjNDU3ZGNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE0VDE3MjIwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJjOTdhZWY4ZGJiNDI5ZDk5YjU1OWY3OTdiOWQ0Y2FlYWI4MzAyZWNmNDAxY2FjYTY1YWRhNDMzODNiMWEzYzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pVryVXm3Vqc2w6B95X-Gjx-vGJPqEETpZHHLWSXxUcQ)
> 
> ### Alt Text: Infographic entitled Community Survey Results
> Long Description: The infographic provides a snapshot of different data points from the GitHub Community Survey (October 2023). It has a solid black background and graphics are accented in purple, two different pink, and yellow.
> 
> Centered at the top of the graphic is the text: "Thank you to the 150+ members who shared their candid feedback with us in our community survey. We enjoyed reading the responses and learning how we can improve!"
> 
> There are four different graphics to correspond with four different data points and descriptions. The graphics and descriptions are as follows:
> 
> A graphic rocket that is yellow, next to a header entitled “Community Usage”, and the text "The most common way you use Community Discussions: learning about new features and updates. Closely followed by report bugs and Search for already submitted feature requests or bugs".
> 
> Below and to the left, a percentage bar that is 55% yellow and 45% pink to represent the data shared: "Value", and the text: "A little more than half of you (55%) found the community helps you get more value out of GitHub and all of you gave us ideas to help that improve!"
> 
> Then, a pink speech bubble with an off-centered yellow outline saying 39% in with paired with the text: "Have been a part of the community for three months or less!" and "If you’re new to the community be sure to check out our New Member Welcome thread to introduce yourself."
> 
> Next, a mostly white calendar with pink and yellow accents and the 26% percent figure is coupled with the header: "26% Said they visit the community everyday or more often." and the text: "This was the most common visit frequency and we’re thrilled so many of you join us daily."
> 
> Below that is the section with the yellow header "In Your Own Words" subheaded with "What you had to say!" and contain positive and negative direct quotes from survey respondents.
> 
> Under the purple header "What You Like..." are the following quotes: “Having a (public) portal to connect to the GitHub developers to give feedback and report bugs without having to open a (private) support ticket.”, “Being able to help others with things I'm passionate about teaching.", and "“The brilliance of its members!!“.
> 
> Under the purple header "Where We Can Improve..." are the following quotes: “More tutorials and more individuals reaching out to new members to share knowledge about how to get more involved & make contributions”, "It often feels like the community feedback is completely ignored”, and "“The feedback system”.
> 
> The last section has a purple heading "Stay tuned" and underneath the text "We run this survey twice a year so we can continuously improve your community experience. Be on the lookout!"
> 
> Alt Title: Infographic for GitHub Community Discussions Survey (October 2023) File Name: Jan 2024 Community Survey Results. png

**Comment by kevintranla1982** [comment]
Y67

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
